README.md

```diff
@@ -61,8 +61,8 @@ There is only an `Octo <object> <action> [arguments]` command:
 | gist | list | [repo] [key=value]*<br>[Available keys](https://cli.github.com/manual/gh_gist_list):  `repo`\|`public`\|`secret`<br>Mappings:<br>`<CR>`: Append Gist to buffer<br>`<C-b>`: Opens Gist in web browser |
 | comment | add | |
 | | delete | |
-| | resolve | |
-| | unresolve | |
+| thread | resolve| Mark a review thread as resolved |
```

Comment:
>

This is another try for a review conversation test.
When I hit the `<space>ca`, the window for this comment buffer opened. But The cursor was in the quickfix window and in insert mode (which cause errors...).
Also the content from the cite above was in the comment buffer initially.
But I can press `<CR>` again. :grimacing:

Reply:
How can I add another comment into the thread here? Is that already possible?

Reply:
> The cursor was in the quickfix window and in insert mode (which cause errors...).

That should not be the case, it should be in the comment window and few two lines below the virtual text "header"

Do you get this behaviour consistently?

> How can I add another comment into the thread here? Is that already possible?

No, You can only place "initial" comments during a review. After the review you can always add replies from the PR buffer of from the thread window in `Octo review threads`

Reply:
> Do you get this behaviour consistently.

Yes, all the time.

> No, You can only place "initial" comments during a review. After the review you can always add replies from the PR buffer of from the thread window in Octo review threads

Yes, I meant from the PR buffer. So I submitted my review with this comment and then reloaded the PR buffer so I could see my own review. Now I wanted to add a comment below my own comment. :grimacing:
How would I do this? :thinking_face:

Reply:
> Yes, all the time.

weird, will take a look

> How would I do this? :thinking_face:

place the cursor somewhere in the middle of the thread, then `<space>ca`

Reply:
> weird, will take a look

I would assume this is kinda related to my issue that the split is broken or does not work as expected. They all stagger on top of each other atm. But it could be a hint that the movement to the comment buffer window should be more precisely with a window ID and not a movement in space. If you know what I mean? But this is just a guess what leads to this issue.

Reply:
#108

Reply:
Oh I can't close the conversation, even I started it.

Reply:
Im currently moving to window handle ids with `nvim_buf_set_current_win`. My guess is that for some reason the split commands I use are not deterministic or you have a different split configuration (splitright, splitbelow, etc.).

For example, if you are in a one window layout and open the qf window, then move to QF win and do `rightbelow vert sbuffer <buffer id>` it should split the qf window vertically in two and then move the cursor to the new window. Is that the case for you?

Reply:
Damn. Yes it gets caused by my own plugin [vim-qfloc-tailor](https://github.com/weilbith/vim-qfloc-tailor). I moves the quickfix/location list windows to the bottom of the screen and adopt the size to the number of entries + one empty line if below a maximum threshold, else limit the size to this maximum of line numbers. This works based on `autocmd BufReadPost quickfix ...`. :thinking:

Reply:
Im concerned other qf plugins may conflict with octo layout so will install it tonite and try to see if there is anything I can do on my end. Any ideas are welcome :)

Reply:
I was able to reproduce the horizontal split (which doesnt look that bad xD) but not the cursor moving into the QF window before moving into insert mode. Did you still get that issue too?

Reply:
> Did you still get that issue too?

Nope. As I suspected these issues were related. So the cursor now lands correctly inside the comment buffer in insert mode. :+1:

Reply:
> Im concerned other qf plugins may conflict with octo layout so will install it tonite and try to see if there is anything I can do on my end. Any ideas are welcome :)

Cool. Thank you very much. I think you can check two of the "bugs" [here](https://github.com/pwntester/octo.nvim/pull/105#issuecomment-787806628).

Reply:
My only idea is to open a new tab with the diff vertical diff view (should be automatic if the user has set it in the `diffopt`), then open the quickfix window and use a floating window to make the comment. :stuck_out_tongue_winking_eye:

Reply:
may use a float win indeed so its more consistent with top level review comment :+1:

Reply:
I just realized that in issue #95 is was working with the correct split despite I had my plugin active. What changed to cause this break since then? :thinking:

Reply:
🤔 not sure, probably too much and I dont feel like doing a bisect if Im going to implement the float window. Also, the float window allow us to have a different window above with the header so no need to use confusing virtual text